flexget/__init__.py

```diff
@@ -38,7 +47,14 @@ def main(args=None):
                     os.path.join(manager.config_base, manager.options.profile),
                 )
             else:
-                manager.start()
+                m = threading.Thread(target=manager.start, daemon=True)
+                m.start()
+                if tray_icon:
+                    while not manager_loaded:
```

Comment:
This will wait forever if we are launching flexget not in daemon mode since the manager.daemon.started event will never fire.

Reply:
fixed by setting a timeout on the event.wait()